### PR TITLE
Add docs for enable_content_listener option.

### PR DESCRIPTION
**Pull request by benglass**
| Q             | A
| ------------- | ---
| Doc fix?      | no
| New docs?     | yes https://github.com/symfony-cmf/SeoBundle/pull/209
| Applies to    | Next release
| Fixed tickets | https://github.com/symfony-cmf/SeoBundle/pull/209

### CODE REVIEW

**Comment by benglass** [comment]
@dbu docs PR for https://github.com/symfony-cmf/SeoBundle/pull/209

File: bundles/seo/configuration.rst [hunk 1]
```diff
@@ -128,3 +128,16 @@ of ``cmf_core.persistence.phpcr.use_sonata_admin``.
 
 The name of the form group of the group provided by the Sonata Admin
 Extension.
+
+``enable_content_listener``
+""""""""""""""
+
+.. versionadded:: 1.2
```
Comment (dbu) [inline bundles/seo/configuration.rst]: i just backported this to 1.1 now, so this needs to be adjusted. can you please close the PR and create a new one against the 1.2 branch of the docs?

File: bundles/seo/configuration.rst [hunk 2]
```diff
@@ -128,3 +128,16 @@ of ``cmf_core.persistence.phpcr.use_sonata_admin``.
 
 The name of the form group of the group provided by the Sonata Admin
 Extension.
+
+``enable_content_listener``
+""""""""""""""
```
Comment (dbu) [inline bundles/seo/configuration.rst]: this needs to be aligned

File: bundles/seo/configuration.rst [hunk 3]
```diff
@@ -128,3 +128,16 @@ of ``cmf_core.persistence.phpcr.use_sonata_admin``.
...
+
+.. versionadded:: 1.2
+    The ``enable_content_listener`` configuration key was introduced in SeoBundle 1.2
+
+**type**: ``boolean`` **default**: ``true``
+
+Whether or not the ``Symfony\Cmf\Bundle\SeoBundle\EventListener\ContentListener`` should be loaded
+as a ``kernel.request`` listener. The ContentListener is responsible for extracting seo data from
```
Comment (dbu) [inline bundles/seo/configuration.rst]: s/seo/SEO

File: bundles/seo/configuration.rst [hunk 4]
```diff
@@ -128,3 +128,16 @@ of ``cmf_core.persistence.phpcr.use_sonata_admin``.
...
+CMF content documents. Set this to false if you want to use your own content listener. You will need
+to register your listener as a service and tag it as a ``kernel.request`` listener.
```
Comment (dbu) [inline bundles/seo/configuration.rst]: not sure about the last sentence. maybe a user just does not want this mechanism.

**Comment by dbu** [comment]
makes me wonder if we mention the listener anywhere else. we should explain how things work in the intro, and have a link to this section. and then from the doc reference link back to the feature description.

**Comment by benglass** [comment]
@dbu Your comments have been addressed although I did not add a new section explaining the content listener. I think that would be a good addition but I'm not sure its 100% necessary for this PR and I don't really have time to do that today.

What branch do you want me to re-open this PR against? I'm not sure based on the ongoing conversations about merging this feature into 1.1 vs 1.2

**Comment by dbu** [comment]
committed to branch 1.2 of the docs in ffa4f4ebab2f5b016b79fcac071b524d46a2edea and fixed versionadded in c667ab3d931f82668ea16197445019ec030b1dfd

**Pull request closed without merging**